test/simplifyExpression/distributeSearch/distributeSearch.test.js

```diff
@@ -47,6 +47,16 @@ describe('distribute', function () {
       ['(-2x^2 * 3x - 2x^2 * -4)',
         '(-6x^3 + 8x^2)']
     ],
+    ['(x - 2)^2',
+      ['(x * (x - 2) - 2 * (x - 2))',
```

Comment:
@ldworkin the special expansion `(a - b)^2 = a^2 - 2ab + b^2` is usually used in this situation. So `(x - 2)^2` would be rewritten to `x^2 -4x + 4`. ([Source](http://www.mathexpression.com/basic-algebra-formulas.html))